runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcSceneOverlay.java

```diff
@@ -176,15 +177,16 @@ private void renderNpcOverlay(Graphics2D graphics, NPC actor, String name, Color
 
 		if (config.drawNames())
 		{
-			Point textLocation = actor.getCanvasTextLocation(graphics, name, actor.getLogicalHeight() + 40);
+			Point textLocation = actor.getCanvasTextLocation(graphics, actor.getName(), actor.getLogicalHeight() + 40);
```

Comment:
Make `Text.removeTags(actor.getName())` a variable and use that for both the `textLocation` and the `OverlayUtil.renderTextLocation` calls.

Reply:
Good catch!